pyproject.toml

```diff
@@ -52,6 +52,7 @@ test = [
     "param",
     "pytest",
     "pytest-benchmark",
+    "holoviews",
```

Comment:
This seems wrong. If we need holoviews for testing, then he have a dashboard test. So shouldn't that use the 'dashboard' extra?

Reply:
I don't think there are tests that use Holoviews, that is why it was not in `test`.

@jl-wynen The `dashboard` extra is to avoid installing GUI bits in the backend. We could still have tests that need (a subset) of what is in that extra, in which case having it in `test` should be fine. No need to install all GUI deps for tests?